docs/docusaurus.config.ts

```diff
@@ -104,35 +101,40 @@ const config: Config = {
           docId: 'api/index',
           position: 'left',
         },
+
+        // Conditionally display version dropdown when in Vercel production environment
+        //
+        //     https://vercel.com/docs/projects/environment-variables/system-environment-variables#VERCEL_ENV
+        //
+        // Otherwise display a link to the latest docs site; this will used in local and preview builds.
+        process.env.VERCEL_ENV && process.env.VERCEL_ENV === 'production'
```

Comment:
@neverett this only shows the drop down menu for `VERCEL_ENV=production`, which means the default local experience of `yarn start` will show a "Latest docs" link.

Would you prefer that we see the drop down in local too?